examples/tests/sdk_tests/queue/push.w

```diff
@@ -0,0 +1,28 @@
+bring cloud;
+
+let q = new cloud.Queue();
+
+test "push" {
+  let obj = Json {
+    k1: 1,
+    k2: "hello",
+    k3: true,
+    k4: {
+        k1: [1, "a", true, {} ]
+    }
+  };
+
+  q.push("Foo");
+  
+  assert(q.approxSize() == 1);
+
+  q.pop();
+  q.push("Bar", "Baz");
+
+  assert(q.approxSize() == 2);
+
+  q.purge();
+  q.push("", "\r", "${obj}");
+
+  assert(q.approxSize() == 3);
```

Comment:
I'm sorry this is failing on tf-aws 😢 ... 
<img width="766" alt="image" src="https://github.com/winglang/wing/assets/39455181/8a57db28-3769-4809-a3aa-22b4f59a0bae">
I think we should use `waitUntil` for each time we check the `q.approxSize()` as it takes time to update in the cloud, (also make sure to extend the test timeout if needed, see: https://github.com/winglang/wing/blob/1c81ccca144c73575ba9f30f4799d3467b2245c4/examples/tests/sdk_tests/queue/timeout.w#L35)